internal/gui/tabbar.go

```diff
@@ -0,0 +1,15 @@
+package gui
+
+import (
+	"github.com/rivo/tview"
+)
+
+func newTabBar(g *Gui) *tview.TextView {
+	return tview.NewTextView().
+		SetDynamicColors(true).
+		SetRegions(true).
+		SetWrap(false)/*.
+		SetHighlightedFunc(func(added, removed, remaining []string) {
```

Comment:
A bug might need to be submitted to `rivo/tview` about this. But until then the workaround I have will suffice.